src/content/docs/infrastructure/host-integrations/host-integrations-list/elasticsearch/elasticsearch-integration.mdx

```diff
@@ -29,6 +29,10 @@ To install the Elasticsearch monitoring integration, run through the following s
 
 Our integration is compatible with Elasticsearch 7.x through 8.X.
 
+### Elasticsearch user
+
+The integration requires an Elasticsearch user with `monitor` privileges in the cluster and all indices. See [elastic's authorization documentation for details](https://www.elastic.co/guide/en/elasticsearch/reference/current/authorization.html).
```

Comment:
Suggested rewording:

The integration requires an Elasticsearch user with `monitor` privileges for the entire cluster and for all indices. For details, see [Elasticsearch user authorization](https://www.elastic.co/guide/en/elasticsearch/reference/current/authorization.html).

Reply:
I'd also call this out on both the Elasticsearch integration topics for the time being. I have create a Jira ticket (https://new-relic.atlassian.net/browse/NR-338527) to fix the duplicate page issue but since that needs a bit of research, I suggest we add the requirements in both topics.